src/lib/create-utils.js

```diff
@@ -85,6 +86,33 @@ exports.enhanceReadmeWithLocalSuffix = async (readmePath, manifestPromise) => {
     }
 };
 
+/**
+ * Goes to the Actor directory and creates INPUT.json file from the input schema prefills.
+ * @param {string} actFolderDir
+ */
+exports.createInputFromSchema = async (actFolderDir) => {
+    try {
+        const currentDir = process.cwd();
+        process.chdir(actFolderDir);
+        const { inputSchema } = await readInputSchema();
+
+        if (inputSchema) {
+            const input = Object.keys(inputSchema.properties).reduce((acc, key) => {
+                const { prefill } = inputSchema.properties[key];
+                if (prefill) acc[key] = prefill;
+                return acc;
+            }, {});
+
+            fs.mkdirSync('./storage/key_value_stores/default', { recursive: true });
```

Comment:
Please, do not use sync functions inside an async function. If the function was sync then the sync stuff is acceptable in CLI.